qpylib/app_qpylib.py

```diff
@@ -9,7 +9,10 @@
 Q_CACHED_MANIFEST = None
 
 def get_app_id():
-    return get_manifest_field_value('app_id', 0)
+    app_id = os.getenv('QRADAR_APP_ID')
+    if app_id is None:
```

Comment:
No need for these 2 lines, just supply 0 as default to os.getenv

Reply:
Also, it's possible that QRADAR_APP_ID is not numeric so you need to do this:
```
try:
    return int(app_id)
except ValueError:
    raise ValueError('Environment variable QRADAR_APP_ID has non-numeric value {0}'.format(app_id)
```
and add a unit test for this scenario.

Reply:
yes it was a string but I was able to put int() around the whole statement including the default to solve it although it would be redundant when the default value is chosen

Reply:
ah nevermind I misread that I see what you are saying

Reply:
os.getenv takes a 2nd arg which is the default to return if the env var is not set, so that's where you supply the 0, not `or 0`.  Your code as it stands will generate ValueError if the env var is not a number.